composer.json

```diff
@@ -68,6 +68,7 @@
         "drupal/redirect": "^1",
         "drupal/reroute_email": "2.1.0",
         "drupal/responsive_preview": "^1.0",
+        "drupal/restui": "^1.20",
```

Comment:
Shoudn't this be added at the module level?

Reply:
Added to headless composer.json